src/ResourceManager/Compute/Commands.Compute/help/Update-AzureRmVmss.md

```diff
@@ -175,7 +176,6 @@ The acceptable values for this parameter are:
 Type: ResourceIdentityType
 Parameter Sets: (All)
 Aliases: 
-Accepted values: SystemAssigned
```

Comment:
?? Seems like AcceptedValues should not be updated here

Reply:
(all the changes below to AcceptedValues seem incorrect)

Reply:
Updated - not sure why that happened.